addons/mail/static/src/messaging/component/followers_menu/followers_menu.scss

```diff
@@ -0,0 +1,40 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_FollowersMenu {
+    margin-right: map_get($spacers, 3);
+    position: relative;
+}
+
+.o_FollowersMenu_dropdown {
+    display: flex;
+    flex-flow: column;
+    padding-top: 0;
+    padding-bottom: 0;
+    overflow-y: auto;
+}
+
+.o_FollowersMenu_followers {
+    display: inline-block;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_FollowersMenu_followButton {
+    color: gray('600');
+}
+
+.o_FollowersMenu_unfollowButton {
+    color: gray('600');
+
+    &.o-following {
+        color: $green;
+    }
+
+    &.o-unfollow {
+        color: $orange;
+    }
```

Comment:
Just asking, don't we want success and warning for consistency? This is specifically green and orange here?

Reply:
yes it's specials colors I think; maybe the green could be success but the orange is not the warning one ; in old code it was specific if I remember well

Reply:
Seems strange for consistency but ok for minimal diff here / not changing the feature.